packages/web/pages/api/broadcast-transaction.ts

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line unused-imports/no-unused-imports
```

Comment:
Why do we need this?

Reply:
lint fails, no idea how to fix it. This did not help.

Reply:
`yarn build` should be run in the `/packages/web` to generate the definition this line imports, otherwise we would get a lint error

Reply:
yes, I was running it from web.

Reply:
reverted and working now.